zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/TestSupport.java

```diff
@@ -18,8 +19,27 @@ public final class TestSupport {
       DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch")
           .withTag(RestClient.class.getPackage().getImplementationVersion());
 
+  private static final DockerImageName OPENSEARCH_IMAGE =
+      DockerImageName.parse("opensearchproject/opensearch").withTag("2.17.1");
+
   private TestSupport() {}
 
+  /**
+   * Returns an OpenSearch container pointing at the same version as the {@link
+   * org.opensearch.client.RestClient}.
+   *
+   * <p>The container is configured to use 512m of heap and 512m of direct memory. This is required
+   * because OpenSearch, by default, will grab all the RAM available otherwise.
+   *
+   * <p>Additionally, security is explicitly disabled to avoid having tons of warning printed out.
+   */
+  @SuppressWarnings("resource")
+  public static OpensearchContainer<?> createDefaultOpensearchContainer() {
+    return new OpensearchContainer<>(OPENSEARCH_IMAGE)
+        .withEnv("OPENSEARCH_JAVA_OPTS", "-Xms256m -Xmx512m -XX:MaxDirectMemorySize=536870912")
```

Comment:
🔧 the comment above says `The container is configured to use 512m of heap and 512m of direct memory` while here I see 256mb 😄